### DISCUSSION TITLE
Has https://wiki.eclipse.org/Tycho/eclipse-repository been migrated?

### DISCUSSION THREAD

**Discussion by vogella** [eclipse-tycho/tycho#4039] [Q&A]
Has https://wiki.eclipse.org/Tycho/eclipse-repository been migrated to the official documentation? I cannot find it. If it has not been migrated, I offer to migrate it, I think it is useful.

**Comment by laeubi** [comment]
We have this documentation here:

https://tycho.eclipseprojects.io/doc/latest/Category.html (sourced from [here](https://github.com/eclipse-tycho/tycho/blob/main/src/site/markdown/Category.md) and https://tycho.eclipseprojects.io/doc/latest/PackagingTypes.html (sourced from [here](https://github.com/eclipse-tycho/tycho/blob/main/src/site/markdown/PackagingTypes.md)) if you feel anything missing please provide a PR to enhance it. There is also many useful information directly in the mojo descriptions.
